crates/volta-migrate/src/v1.rs

```diff
@@ -87,17 +87,18 @@ impl TryFrom<V0> for V1 {
 
             debug!("Removing old Volta binaries");
             let old_volta_bin = new_home.root().join("volta");
-            if old_volta_bin.exists() {
-                remove_file(&old_volta_bin).with_context(|| ErrorKind::DeleteFileError {
+
+            remove_file(&old_volta_bin)
+                .with_context(|| ErrorKind::DeleteFileError {
                     file: old_volta_bin,
-                })?;
-            }
+                })
+                .accept_error_as_default_if(|err| err.is_not_found_error_kind())?;
 
             let old_shim_bin = new_home.root().join("shim");
-            if old_shim_bin.exists() {
-                remove_file(&old_shim_bin)
-                    .with_context(|| ErrorKind::DeleteFileError { file: old_shim_bin })?;
-            }
+
+            remove_file(&old_shim_bin)
+                .with_context(|| ErrorKind::DeleteFileError { file: old_shim_bin })
+                .accept_error_as_default_if(|err| err.is_not_found_error_kind())?;
```

Comment:
We actually have a helper in `volta_core::fs` named `remove_file_if_exists`, which explicitly doesn't error if the file doesn't exist, so I think that's what we want here.

Reply:
Done